src/AVFoundation/AVCompat.cs

```diff
@@ -169,6 +169,22 @@ public AVAudioUnitComponentManager ()
 		}
 	}
 
+	public delegate int AVAudioSourceNodeRenderHandler (bool isSilence, AudioToolbox.AudioTimeStamp timestamp, uint frameCount, ref AudioToolbox.AudioBuffers outputData);
+
+	partial class AVAudioSourceNode {
+		[Obsolete("Use 'AVAudioSourceNode (AVAudioSourceNodeRenderHandler2)' instead.")]
+		public IntPtr Constructor (AVAudioSourceNodeRenderHandler renderHandler)
```

Comment:
missed it earlier but that's incorrect - it's how we bind constructors
however for manual bindings you need to create _normal_ constructor

```csharp
public AVAudioSourceNode (AVAudioSourceNodeRenderHandler renderHandler)
```

API diff complains about it